docker-compose.dev.yml

```diff
@@ -29,9 +29,31 @@ services:
       - "traefik.http.services.traefik.loadbalancer.server.port=8080"
 
   db:
-    image: postgres:14.0-alpine
+    image: postgres:15.0-alpine
```

Comment:
FYI, I added a config file to simplify this: `./scripts/postgresql.conf`. You'll need to rebase and fix conflicts. I'm fine with using the command line rather than config file but the current overrides need to be backported.

Reply:
Done!